README.md

```diff
@@ -33,3 +33,19 @@ Below are some ideas I'll implement in the comming days, no promise to respect t
 -   [ ]  Slack chatbot connector
 -   [ ]  Design plugin to make the chatbot extensible
 -   [ ]  Some plugins out of the box
+
+------  
+to run the shell script called `set_telegram_webhook.sh` following steps might be needed:  
```

Comment:
Add a title here like: 
How to run the example.

Reply:
thanks for handling it.